versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.71"
+      version = ">= 4.42.0"
```

Comment:
Since this is now a breaking change, lets just go all the way up to v5.0 of the AWS provider

Reply:
sounds good!

Reply:
ah missed your second comment